src/core/request/query_builder.ts

```diff
@@ -108,12 +108,19 @@ export class KairosDBQueryBuilder {
         return this.url + this.apiPath + urlStub;
     }
 
-    private buildTagsRequestBody(metricName, filters = {}) {
+    private buildTagsRequestBody(metricName, filters = {}): any {
+        if (this.templateSrv.timeRange) {
+            return {
+                cache_time: 0,
+                metrics: [{name: metricName, tags: filters}],
```

Comment:
Minor: Since we are using the same data, changing only two fields, I think that approach with defining the basic object and updating it will also work.
Like this:
```
const result: any = {
  cache_time: 0,
  metrics: [{ name: metricName, tags: filters }],
  start_absolute: 0
};
if (this.templateSrv.timeRange) {
  result.start_absolute = this.templateSrv.timeRange.from.unix() * 1000;
  result.end_absolute = this.templateSrv.timeRange.to.unix() * 1000;
}
return result;
```